core/src/main/java/pl/skidam/automodpack_core/protocol/netty/NettyServer.java

```diff
@@ -169,9 +179,7 @@ public boolean stop() {
                 serverChannel = null;
             }
 
-            if (shouldHost) {
                 shouldHost = false;
```

Comment:
```suggestion
            shouldHost = false;
```
I forgot to change the indentation of this line.